docs/contents/pdf_model.puml

```diff
@@ -150,8 +150,8 @@
             **content**: container for elements of type
             ""Chapter/Table/Figure/Paragraph""
 
-            A chapter may not contain a HoritontalBox, 
-            when the chapter is a virtual one.
+            A chapter does not contain a HoritontalBox, 
```

Comment:
Typo: HorizontalBox
I would also rephrase to:

HorizontalBox is `None`, when the search algorithm did not find a suitable text box on the page (aka. ghost chapter).